oci_python_image/py_image_layer.bzl

```diff
@@ -51,11 +59,18 @@ def py_image_layer(name, binary, root = None, **kwargs):
         srcs = ["%s/app/runfiles" % name],
         **pkg_tar_kwargs
     )
+    
+    pkg_tar(
+        name = "%s/python" % name,
+        srcs = ["%s/python/runfiles" % name],
+        **pkg_tar_kwargs
+    )
 
     native.filegroup(
         name = name,
         srcs = [
             "%s/app" % name,
+            "%s/python" % name,
```

Comment:
Does the final output image respect this layer order? If so, we should probably move the "python interpreter" layer to the beginning, since the "app" layer will (in most cases) change with higher frequency.